sources/packages/backend/apps/test-db-seeding/src/db-seeding/aest/basic-seeding/create-cas-distribution-accounts.models.ts

```diff
@@ -0,0 +1,61 @@
+import { OfferingIntensity } from "@sims/sims-db";
+
+export interface CASDistributionAccountBaseData {
+  awardValueCode: string;
+  offeringIntensity: OfferingIntensity;
+  operationCode: "CR" | "DR";
+}
+
+export const CAS_DISTRIBUTION_ACCOUNTS_INITIAL_DATE: CASDistributionAccountBaseData[] =
```

Comment:
It is great idea to have it data seeded.

Reply:
The idea is to share them with the API E2E tests also in upcoming PRs in this sprint.